viewer/public/common.js

```diff
@@ -808,6 +808,42 @@ $(document).ready(function() {
     });
     return false;
   });
+  
+  //////////////////////////////////////////////////////////////////////////////////
+  // Reports
+  //////////////////////////////////////////////////////////////////////////////////
+  $("#queryReport").click(function (e) {
```

Comment:
when more then one report exists, only the first one can be used
ex: report2, report3 can't be used
report
--report1
--report2
--report3

recommend:
$(".queryReport").click(function (e) {